.github/workflows/selfdrive_tests.yaml

```diff
@@ -210,9 +210,9 @@ jobs:
       run: |
         ${{ env.RUN }} "scons -j$(nproc)"
     - name: Run replay
-      timeout-minutes: 30
+      timeout-minutes: 2
       run: |
-        ${{ env.RUN }} "coverage run selfdrive/test/process_replay/test_processes.py -j$(nproc) && \
+        ${{ env.RUN }} "FILEREADER_CACHE=1 coverage run selfdrive/test/process_replay/test_processes.py -j$(nproc) && \
```

Comment:
isn't this set globally?

Reply:
yep in env.RUN. why isn't it the default though?